src/assets/scss/custom.scss

```diff
@@ -9,8 +9,8 @@ $error: #eb5757;
 $carousel-control-width: 3%;
 
 @for $i from 1 through 6 {
-  $zIndexBackdrop: #{1000 + (5 * $i)};
-  $zIndexContent: #{1000 + (5 * $i) + 2};
+  $zIndexBackdrop: #{1048 + $i * 2};
+  $zIndexContent: #{1048 + $i * 2 + 1};
   .modal-backdrop.show:nth-of-type(#{$i}) {
```

Comment:
Can you add a comment why this constant 1048?

Reply:
Done

Reply:
Thanks. How about something like this to make it more obvious:
```css
// default z-index for modal backdrops in bootstrap
$zIndex: 1050;
@for $i from 1 through 6 {
  .modal-backdrop.show:nth-of-type(#{$i}) {
    z-index: $zIndex;
  }
  $zIndex: $zIndex + 1;
  div[role="dialog"][aria-modal="true"]:nth-of-type(#{$i}) {
    z-index: $zIndex;
  }
  $zIndex: $zIndex + 1;
}
```

Reply:
Done. The backdrop's default is actually 1040, and the content is 1050, so I bumped it back to 1040 to make sure it doesn't break anything just in case, but I tested it and it looks fine